Classes/Task/BatchResizeAdditionalFieldProvider.php

```diff
@@ -121,13 +123,21 @@ public function validateAdditionalFields(array &$submittedData, \TYPO3\CMS\Sched
             $directoryConfig = FAL::getDirectoryConfig($directory);
             if (!@is_dir($directoryConfig['basePath'] . $directoryConfig['directory'])) {
                 $result = false;
-                (new fakeSchedulerModuleController())->addMessage(
+
+                $message = GeneralUtility::makeInstance(
+                    FlashMessage::class,
                     sprintf(
                         $GLOBALS['LANG']->sL('LLL:EXT:image_autoresize/Resources/Private/Language/locallang_mod.xlf:msg.invalidDirectories'),
                         $directory
                     ),
-                    FlashMessage::ERROR
+                    'EXT:image_autoresize: invalidDirectories error',
+                    ContextualFeedbackSeverity::ERROR,
```

Comment:
`ContextualFeedbackSeverity` is introduced in TYPO3 v12, thus it breaks compatibility with TYPO3 v10 and TYPO3 v11.

Reply:
Yes, can be done, i leave that up to you :-)